config/reports/lpdcBestuurseenheidReport.js

```diff
@@ -19,25 +19,35 @@ export default {
       PREFIX skos:    <http://www.w3.org/2004/02/skos/core#>
       PREFIX adms:    <http://www.w3.org/ns/adms#>
       PREFIX dct:     <http://purl.org/dc/terms/>
-      PREFIX schema:  <http://schema.org/> 
+      PREFIX schema:  <http://schema.org/>
       PREFIX pav:     <http://purl.org/pav/>
+      PREFIX foaf:    <http://xmlns.com/foaf/0.1/>
 
-      SELECT DISTINCT ?uriBestuurseenheid ?naam ?typeUri ?type ?uriPublicService ?title ?modified ?status ?statusLabel WHERE {
+      SELECT DISTINCT ?uriBestuurseenheid ?naam ?typeUri ?type ?uriPublicService ?title ?modified ?modifiedBy ?status ?statusLabel WHERE {
         ?uriPublicService
           a                     lpdcExt:InstancePublicService ;
             adms:status           ?status ;
             schema:dateModified   ?modified ;
             dct:title             ?title ;
             pav:createdBy         ?uriBestuurseenheid .
 
-        ?uriBestuurseenheid 
+        ?uriBestuurseenheid
           a                     besluit:Bestuurseenheid ;
             skos:prefLabel        ?naam ;
             besluit:classificatie ?typeUri .
 
         ?typeUri skos:prefLabel ?type .
 
         ?status skos:prefLabel ?statusLabel .
+
+        OPTIONAL {
```

Comment:
I did an optional here just in case, since not every instance will have a lastModifiedBy right away